build_local_windows.ps1

```diff
@@ -52,7 +52,15 @@ python -m venv "$tmpdir/dcos_build_venv"
 . "$tmpdir/dcos_build_venv/Scripts/Activate.ps1"
 
 # Install the DC/OS tools
-./prep_local_windows.ps1
-
+#./prep_local_windows.ps1
+./prep_teamcity_windows.ps1
 # Build a release of DC/OS
 release create $env:USERNAME local_build windows
```

Comment:
@pcrews & @orsenthil should we use here something else like commit or pr number?

Reply:
PR number or branch name will be better.

Reply:
returned the main `$config_yaml` back

Reply:
> returned the main $config_yaml back


What does this mean?

Reply:
@DominikDary asked to keep $config_yaml which stores interim artifacts into Azure/AWS storage. It may be redundant, but in this case we had to return previous $config_yaml and make upload to the storage using "release create " app.
For PROD and Client's use there is Installer Tar ball upload using PS1 Cmd-Lets, that what we agreed on before